Changes

```diff
@@ -336,8 +336,12 @@ Release branch for 4.06:
 - MPR#7631, GPR#1355: "-linscan" option crashes ocamlopt
   (Xavier Clerc, report by Paul Steckler)
 
-- MPR#7642: on ARMv4 and ARMv5, wrong register allocation for integer multiply
-  (Xavier Leroy, report by Stéphane Glondu and Mantis user "infinity0")
+- MPR#7642, GPR#1411: ARM port: wrong register allocation for integer
+  multiply on ARMv4 and ARMv5; possible wrong register allocation for
+  floating-point multiply and add on VFD and for floating-point
+  negation and absolute value on soft FP emulation.
+  (Xavier Leroy, report by Stéphane Glondu and Mantis user "infinity0",
```

Comment:
You should use single-quotes instead of double-quotes.
cc @gasche

Reply:
I will make a pass to translate all known usernames (only users that wish to remain pseudonymous are kept as pseudonyms) later, so it does not matter -- I don't want to draw Xavier's ire by being too picky about his Changes entries. According to his github page, "infinity0" goes by the name Ximin Luo.

Reply:
Thank you for your leniency concerning name quoting, much appreciated.